schemas/src/main/java/com/scottlogic/deg/schemas/v3/ConstraintDTO.java

```diff
@@ -40,4 +38,23 @@
     /** the constraint to apply if 'if_' is false */
     @JsonProperty("else")
     public ConstraintDTO else_;
+
+    private String is;
+
+    @JsonIgnore
+    private boolean hasIsConstraintBeenSet = false;
+
+    @JsonSetter("is")
+    public void setIs(String is) {
+        this.is = is;
+        this.hasIsConstraintBeenSet = true;
+    }
+
+    @JsonGetter("is")
+    public String getIs() throws InvalidSchemaException {
+        if(this.hasIsConstraintBeenSet && this.is == null) {
+            throw new InvalidSchemaException("Cannot recognise is null");
```

Comment:
You could throw the `InvalidProfileException` type here, to save having 2 different exception types.